### PR TITLE
Implements iron-a11y-announcer

### PR DESCRIPTION
**Pull request by cdata**
In order to announce text on screen readers, you need to have a "visible" element in the DOM that changes text content in a specific way. This element is intended to be an a11y announcer on behalf of all other elements that may want to announce text via a screen reader.

/cc @notwaldorf @robdodson

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  <title>iron-a11y-announcer demo</title>
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <link rel="import" href="../../paper-styles/paper-styles.html">
+  <link rel="import" href="x-announces.html">
+
+  <link rel="stylesheet" href="../../paper-styles/demo.css">
+
+  <style is="custom-style">
```
Comment (morethanreal) [inline demo/index.html]: Is there a style in `paper-style` that can be used for this?

Comment (cdata) [inline demo/index.html]: There might be now. :)

**Comment by cdata** [comment]
@yky @notwaldorf PTAL

**Comment by morethanreal** [comment]
this is awesome. lgtm

**Pull request merged**